.github/workflows/ci.yml

```diff
@@ -179,5 +179,6 @@ jobs:
         working-directory: packages/test
 
       - name: Test installation
-        run: npm test
+        run: npx rescript -h && npx rescript build && cat src/Test.bs.js
```

Comment:
Sorry can't point to line 152 that introduces `test:` so writing here: should that be called installation test or something?

Reply:
Done